index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="assets/utiles.css">
+    <title>Event Website</title>
+</head>
+
+<body>
+    <header class="header">
+
+        <!-- Desktop menu -->
+        <div class="header-top">
+            <ul>
+                <li><a href=""><img src="assets/icones/1.png" alt="github" width="30" height="30"></a></li>
+                <li><a href=""><img src="assets/icones/4.png" alt="tweeter" width="30" height="30"></a></li>
+                <li><a href="">Login</a></li>
+                <li><a href="">Sign Up</a></li>
+
+            </ul>
+
+        </div>
+        <div class="desktop-menu">
+            <a href="index.html"><img src="assets/images/GE.png" alt="" width="120" class="logo"></a>
+            <nav>
+                <ul class="desktop-nav">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="">Program</a></li>
+                    <li><a href="">Join</a></li>
+                    <li><a href="">Support</a></li>
+                    <li><a href="">News</a></li>
+                    <li class="ge-campaigne"><a href="">GE campaign</a></li>
+
+                </ul>
+            </nav>
+
+        </div>
+        <button class="menu-button">
+            <img src="assets/icones/menu.png" alt="menu" width="45" height="45">
+        </button>
+
+        <div class="nav-active">
+            <button class="menu-button">
+                <img src="assets/icones/cancel.png" alt="close menu" width="40" height="40">
+            </button>
+            <ul class="list-nav">
+                <li><a href="index.html" class="nav-list">Home</a></li>
+                <li><a href="about.html" class="nav-list">About</a></li>
+                <li><a href="">Program</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Support</a></li>
+                <li><a href="">News</a></li>
+            </ul>
+
+        </div>
+    </header>
+    <section class="head-line-section">
+        <div class="head-line-container">
+            <h2 class="orange">"Hello sharing world"</h2>
+            <h1 class="big-title">GOLDEN EVENT WEBSITE </h1>
+
```

Comment:
### Your Title
![big-title](https://user-images.githubusercontent.com/93039359/211143008-e4b597d4-6a12-4ae5-9042-a4a3aad5850a.PNG)

### Design Title
![big-title](https://user-images.githubusercontent.com/93039359/211143016-9c336707-e9eb-44a5-ad9d-1f915d63cd65.PNG)

- [ ] Your title is not using the correct form which is COCOGOOSE. Kindly use COCOGOOSE font for this title to follow the design 👍